src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -35,6 +34,10 @@
  */
 final class ApiLoader extends Loader
 {
+    /**
+     * @deprecated since version 2.0.10, to be removed in 3.0.
+     *             Use {@see RouteNameGenerator::ROUTE_NAME_PREFIX} instead.
```

Comment:
I would merge this patch in 2.1, WDYT? I don't like to introduce deprecations in patch releases.